client/app/get-cats-handler.js

```diff
@@ -0,0 +1,38 @@
+import { App } from './app.js';
+
+async function getAllCats() {
+
+    const response = await fetch('http://localhost:8080/api/api')
```

Comment:
Hey @oguzkarademir I believe that here you should call the route `allEntries` or `data` on the controllers. On the way that is written just work in local, but it will not work on production ;)